src/js/edu-benefits/1990e/config/form.js

```diff
@@ -116,6 +128,72 @@ const formConfig = {
     educationHistory: {
       title: 'Education History',
       pages: {
+        educationHistory: {
+          path: 'education-history',
+          title: 'Education History',
+          initialData: {
+          },
+          uiSchema: {
+            highSchoolOrGedCompletionDate: date.uiSchema('When did you earn your high school diploma or equivalency certificate?'),
+            'view:educationHistory': {
+              'ui:title': 'Please list any post-high school trainings you have completed.'
```

Comment:
You shouldn't need a separate empty field for this.

You can use the `ui:description` field on `postHighSchoolTrainings` and it should show up with the right spacing:

```js
'ui:description': <h5>Please list any post-high school trainings you have completed.</h5>
```

(You would also need to import React at the top and change the file extension here to jsx, because of the h5 tags)

Not sure what @goldenmeanie thinks, but this instruction reads more like a description than a title, so you could also leave off the `<h5>` tags. And you can also remove the `hideTitle` property to show the "Education after high school" text as a header, potentially.